### DISCUSSION TITLE
OTP Expiration period

### DISCUSSION THREAD

**Discussion by Kushwaha2021** [saran-surya/email_auth#37] [Q&A] [answered]
why there is no time  limit for otp validation

**Comment by saran-surya** [comment, marked as answer]
Hi @Kushwaha2021 As you are handling the process for the OTP process, You can add in your own logic for it. It is designed that we don't have a complex package, And this will have all the base components and we want to have a better reliable option. 

Also I am open to PR regarding the same if you can help out 🙌🚀
